charts/tempo-distributed/templates/metrics-generator/deployment-metrics-generator.yaml

```diff
@@ -101,7 +101,7 @@ spec:
             name: {{ template "tempo.fullname" . }}
           name: tempo-conf
         - name: wal
-          emptyDir: {}
+          emptyDir: {{- toYaml .Values.metricsGenerator.walEmptyDir | nindent 12 }}
```

Comment:
Should this be an emptyDir to begin with? This is the place where the WAL is stored. 🤔 
@annanay25

Reply:
Does this need persisting? Or does it just regenerate it on startup?

Reply:
@mdisibio Martin, can you shine a light on this?

Reply:
Hi, sorry for the delayed response here.

The metrics-generator can generate the WAL directory when it starts up, so if it doesn't exist it will be created anyway, if it already exists it just checks if it's writeable.
So either is fine for this PR 🙂

Reply:
@kvrhdn Does that mean that persistence is not needed? And if so, should this be called a WAL ?